lib/streamlit/caching/cache_utils.py

```diff
@@ -38,11 +42,23 @@
 _LOGGER = get_logger(__name__)
 
 
+@define
+class MsgData:
+    delta_type: str
+    message: Message
+
+
+@define
+class CachedValue:
```

Comment:
CachedValue is an awkward name, since the point is that it's the value _plus_ some message stuff

Reply:
Oh, maybe `CachedResult`